src/core/utils.jl

```diff
@@ -183,14 +184,37 @@ function dictkeys_tosymbols(d::Dict)
             end
         end
         if k in [
-            "coincident_peak_load_active_timesteps"
+            "coincident_peak_load_active_time_steps"
         ]
             try
                 v = convert(Vector{Vector{Int64}}, v)
             catch
                 @warn "Unable to convert $k to a Vector{Vector{Int64}}"
             end
         end
+        if k in [
+            "outage_start_time_steps", "outage_durations"
+        ]
+            try
+                v = convert(Array{Int64, 1}, v)
+            catch
+                @warn "Unable to convert $k to a Array{Int64, 1}"
+            end
+        end
+        if k in [
```

Comment:
I think that all of these converts could be removed by changing their type to Real in the struct definitions.

Reply:
it looks like you commented out these lines for converting scalars to Float64 (that are now handled as Real types). Can we remove the lines?

Reply:
The commented out code was removed in 3c364b32ce52e2637e1528a22c99aa224ed5203c. Apologies for not linking before!